package.json

```diff
@@ -7,11 +7,11 @@
     "test": "NODE_ENV=test sequelize db:migrate:undo:all && npm run db:migrate:test && npm run db:migrate:seed:test && NODE_ENV=test nyc mocha --require @babel/register --timeout 30000 src/tests/index.js",
     "cover": "npm test && nyc report --reporter=text-lcov | coveralls",
     "coveralls": "cat coverage/lcov.info | coveralls",
-    "start": "npm run db:migrate && babel-node src/index.js",
-    "dev": "nodemon src/index.js --exec babel-node --presets @babel/preset-env",
+    "start": "concurrently \"npm run db:migrate && npm run db:seed\" \"babel-node src/index.js\"",
```

Comment:
Do we need this?

Reply:
yes we need these because we need to run multiple commands concurrently and skip other when fail